src/main/webapp/WEB-INF/web.xml

```diff
@@ -46,8 +46,18 @@
         </init-param>
     </filter>
 
+    <filter>
+        <filter-name>XsltFilter</filter-name>
+        <filter-class>com.rackspace.feeds.feedscatalog.ResolveHostFilter</filter-class>
+    </filter>
+
     <filter-mapping>
         <filter-name>Xml2JsonFilter</filter-name>
         <url-pattern>/*</url-pattern>
     </filter-mapping>
+
+    <filter-mapping>
+        <filter-name>XsltFilter</filter-name>
+        <url-pattern>/*</url-pattern>
+    </filter-mapping>
```

Comment:
The ordering is correct.  The resolve filter happen first, then the xml2json filter.